src/messages/css.js

```diff
@@ -1,17 +1,15 @@
-import { oneLine } from 'common-tags';
-
 import { i18n } from 'utils';
 
 export const CSS_SYNTAX_ERROR = {
   code: 'CSS_SYNTAX_ERROR',
   // This will be overriden by the reason passed from the error.
   message: i18n._('A CSS syntax error was encountered'),
-  description: i18n._(oneLine`An error was found in the CSS file being
+  description: i18n._(`An error was found in the CSS file being
     processed as a result further processing of that file is not possible`),
 };
```

Comment:
@willdurand I also wonder if we should manually turn these strings to be actually one line as part of removing the `oneLine` tag. Not sure if that matters in practice (e.g. if not making those one line strings messes with how the table in the text format is formatted in the console).

Reply:
These strings are wrapped at 80chars, that's why it uses backticks (and I kinda prefer that)